core/src/impl/Kokkos_Serial_Task.hpp

```diff
@@ -147,9 +151,11 @@ class TaskQueueSpecializationConstrained<
 
     task_base_type* const end = (task_base_type*)task_base_type::EndTag;
 
-    Impl::HostThreadTeamData* const data = Impl::serial_get_thread_team_data();
+    execution_space serial_execution_space;
```

Comment:
```suggestion
    auto const& serial_execution_space = scheduler.get_execution_space();
```

Reply:
No that's wrong. That won't compile because here we are using a different type of scheduler that does not have `get_execution_space()`. If you look at the OpenMP code, you can see that they use the singleton

Reply:
OK